### ISSUE TITLE
Visibility?

### ISSUE THREAD

**Issue by caltrop-3** [linuxmint/blueberry#67]
can not turn off visibility

more of a bluetooth on/off than a configuration tool

**Comment by BrookeDot** [comment]
I agree that this would be a great addition to have a toggle for "public visibility" I often want to hide my device name when using my laptop out in public.

I found a way to do this manually with:
`sudo hciconfig hci0 noscan`

You can get your own  hciconfig ID by running:
`sudo hciconfig -a`

More info and how to re-enable[ in this answer](https://askubuntu.com/questions/450698/how-to-turn-on-and-off-bluetooth-visibility-mode) on StackExchange.

**Comment by Xmetalfanx** [comment]
I agree that having an "off/on" discover-ability (without turning BT off completely) would be something I'd love to see added

**Comment by besworks** [comment]
This seems like it's related to a bug in gnome-bluetooth that was fixed recently.

https://bugs.launchpad.net/ubuntu/+source/gnome-bluetooth/+bug/1791405

Leaving discoverablity turned on is a pretty big security hole so this should really be addressed ASAP.

**Comment by miniskipper** [comment]
This also bugged me. I have modified it so the adapter is only visible for 60 seconds from start of manager. Also clicking the header icon or the appropriate option in the tray makes the adapter visible for 60 seconds.

![image](https://user-images.githubusercontent.com/2269291/87183668-d4ff7700-c2e6-11ea-9da3-295813c1230f.png)

I would open a PR if desired.

Translation is a TODO

**Comment by mtwebster** [comment]
Do you guys leave the blueberry window open? The adapter only remains discoverable while that is running - at least this is true in Mint 20. I confirm this with both btmgmt->info command and checking with my phone. I'm hesitant to limit that because this is typical behavior in my experience.

It's really intended only as a configuration tool, so it would be limited in its duration. I'd like to hear some use cases that argue otherwise.

Thanks
